manifests/g/GoLang/Go/1.18.0/GoLang.Go.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GoLang.Go
+PackageVersion: 1.18.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://dl.google.com/go/go1.18.windows-amd64.msi
+  InstallerSha256: 57cba0010ce3bc8012cfa68ab0e11181ccd8493a66b109a5b86998e5d2b38ae1
+  UpgradeBehavior: install
+  ProductCode: '{E5DC7A79-C9A0-4472-8159-9F331CAB8E81}'
```

```suggestion
  ProductCode: '{F307E329-805A-4C79-BAEC-7FB35F3FE64B}'
```
winget-create or YamlCreate.ps1 is recommended to make new packages or new versions.

Both of these will update the ProductCode for you. ProductCode is unique for each release of the Go installer and cannot be the same.